data/sound-strings/ia.json/localize.json

```diff
@@ -1,5 +1,108 @@
 {
-"exit": "Exit",
-"you_have_reached_the_destination": "You have arrived",
-"then": "Then"
```

Comment:
Language `ia` has only two strings translated `[exit]` and `[you_have_reached_the_destination]` (are they really translated?). All other strings are from English. Is it correct? Should we keep `ia` language?